### ISSUE TITLE
Invalid variable in ASN1_UTCTIME.get_datetime

### ISSUE THREAD

**Issue by mcepl** [mcepl/M2Crypto#1]
Originally https://github.com/M2Crypto/M2Crypto/issues/2 by @hcastilho:
```python
if month not in self._ssl_months:
    raise ValueError("Invalid date %s: Invalid month: %s" % (date, m))
```

**Comment by mcepl** [comment]
Filed as https://gitlab.com/m2crypto/m2crypto/issues/46

**Comment by mcepl** [comment]
Fixed in 62191971
